android/app/src/main/java/platis/solutions/smartcarmqttcontroller/Model/EmergencyContact.java

```diff
@@ -0,0 +1,70 @@
+package platis.solutions.smartcarmqttcontroller.Model;
+
+public class EmergencyContact {
+
+    private int id;
+    private String first_name;
```

Comment:
When you create new varibles  can you have underscores to seperate variable names? 
For example should it not be firstName, not first_name?